q2_phylogeny/tests/test_raxml.py

```diff
@@ -0,0 +1,186 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import unittest
+import skbio
+import tempfile
+
+from qiime2.plugin.testing import TestPluginBase
+from qiime2.util import redirected_stdio
+from q2_types.feature_data import AlignedDNAFASTAFormat
+
+from q2_phylogeny import raxml
+from q2_phylogeny._raxml import run_command
+
+
+class RaxmlTests(TestPluginBase):
+
+    package = 'q2_phylogeny.tests'
+
+    def test_raxml(self):
+        """Test that output tree is made.
```

Comment:
Thanks for documenting these tests! I haven't seen docstrings used before for this before (as opposed to a plain-old-comment, `#`-style), but am open either way. I want to hear from @ebolyen though, in case this might have some undesired consequence later on.

Reply:
Ahh, good point. I did not notice the lack of docstrings elsewhere. I have no problem using regular comments.